src/prefect/engine.py

```diff
@@ -224,10 +225,20 @@ async def create_then_begin_flow_run(
                 message="Validation of flow parameters failed with error:"
             )
 
+    if flow.flow_run_name:
+        sig = inspect.signature(flow.fn)
+        for kwarg, val in sig.parameters.items():
+            if val.default is not val.empty:
+                parameters.setdefault(kwarg, val.default)
```

Comment:
This block shouldn't be needed, when we get the `parameters` for a flow call we include defaults

https://github.com/PrefectHQ/prefect/blob/name-ur-run/src/prefect/utilities/callables.py#L42

Reply:
The test was failing before I included this though

Reply:
Interesting, I'll check my understanding.